src/components/Hero.js

```diff
@@ -25,25 +27,93 @@ function Hero({handleLogIn}) {
           <MarketingButton primary onClick={handleLogIn}>
             Login
           </MarketingButton>
-          <a href="">
-            <MarketingButton>
-              <Octicon verticalAlign="middle" icon={getIconByName("book")} />
-              {" "}Read The Story
-            </MarketingButton>
-          </a>
+          <MarketingButton primary onClick={handleLogIn}>
+            Signup
+          </MarketingButton>
         </Wrapper>
       </SubHero>
       <TrustSection>
-        {/*
-        <small>Trusted By</small>
-        <ul>
-          <li>ONE</li>
-          <li>TWO</li>
-          <li>THREE</li>
-          <li>FOUR</li>
-        </ul>
-        */}
+        <Wrapper>
+          <small>Trusted By</small>
+          <ul>
+            <li>GraphQL</li>
+            <li>NextJS</li>
+            <li>Electron</li>
+            <li>Golang</li>
+          </ul>
+        </Wrapper>
       </TrustSection>
+      <QuoteSection>
+        <Wrapper>
+          <Link to="/">
+            <img style={{height: 50}} alt="open sauced" src={logo} />
+          </Link>
+          <Flex className="description">
+            Open Sauced provides structure onboarding for new contributors to open source. This structure provides a way to track your next contributions by leveraging a unique dashboard built on top of the GitHub GraphQL API.
+          </Flex>
+          <Flex style={{marginTop: 100}}>
+            <Flex className="quote">
+              <img alt="open sauced" src={"https://github.com/yg.png"} />
+              <span>
+                <q>Dictum fusce ut placerat orci nulla pellentesque dignissim.</q>
+                <Flex className="author">- bdougie</Flex>
+              </span>
+            </Flex>
+            <Flex className="quote">
+              <img alt="open sauced" src={"https://github.com/yg.png"} />
+              <span>
+                <q>Dictum fusce ut placerat orci nulla pellentesque dignissim.</q>
+                <Flex className="author">- bdougie</Flex>
+              </span>
+            </Flex>
+          </Flex>
+        </Wrapper>
+      </QuoteSection>
+      <DetailsSection>
+        <Wrapper>
+          <Flex style={{justifyContent: "space-between"}}>
+            <Flex>
+              <img alt="open sauced" src={"https://github.com/bdougie.png"} />
+              <FlexColumn>
+                <span>Dictum fusce ut placerat orci nulla pellentesque dignissim.</span>
```

Comment:
```suggestion
                <h2>Dictum fusce ut placerat orci nulla pellentesque dignissim.</h2>
```

Should be a header and not a span here.